src/main/io/ledstrip.h

```diff
@@ -92,15 +92,15 @@ typedef enum {
 } colorId_e;
 
 typedef enum {
-    LED_MODE_ORIENTATION = 0,
-    LED_MODE_HEADFREE,
-    LED_MODE_HORIZON,
-    LED_MODE_ANGLE,
-    LED_MODE_MAG,
-    LED_MODE_BARO,
-    LED_SPECIAL,
-    LED_AUX_CHANNEL
-} ledModeIndex_e;
+    LED_FCSTATE_ORIENTATION = 0,
+    LED_FCSTATE_HEADFREE,
+    LED_FCSTATE_HORIZON,
+    LED_FCSTATE_ANGLE,
+    LED_FCSTATE_MAG,
+    LED_FCSTATE_BARO,
```

Comment:
`#define LED_FCSTATE_COUNT          (LED_FCSTATE_BARO + 1)`

Reply:
OK, though I don't understand much of the handling of the `LED_FCSTATE_AUX_CHANNEL` FC State...